### PR TITLE
Fix test bug 291759

### PR DESCRIPTION
**Pull request by dazavala**
Modify the SpringBoot TemplateTest to be less sensitive to Windows file locking.

### CODE REVIEW

**Comment by dazavala** [comment]
#build

File: dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/TemplateTests.java [hunk 1]
```diff
@@ -61,34 +59,44 @@ public static void tearDown() throws Exception {
...
     public void testCreateServerFromSpringBoot2() throws Exception {
         expectedFeatures = new String[] { "servlet-4.0", "ssl-1.0", "transportSecurity-1.0", "websocket-1.1", "springBoot-2.0" };
+        serverName = "com.ibm.ws.springboot.support.fat.TemplateServer2";
+        serverPath = installPath + "/usr/servers/" + serverName;
 
         createServerFromTemplate("springBoot2", expectedFeatures);
     }
 
     void createServerFromTemplate(String templateName, String[] expectedFeatures) throws Exception {
+        cleanupServer();
```
Comment (sebratton) [inline dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/TemplateTests.java:98]: I don't get why you need to do this begore and after for this test sice you're using different servernames. But it seems harmless

**Review by sebratton — APPROVED** [review]

**Comment by dazavala** [comment]
Thanks for the review, @sebratton. The PB indicates everything works as expected. Before merging I investigated a couple concerns regarding these changes: 

1. The test framework does not collect server content for servers created using simplicity's liberty server utils — i.e. deleting these test servers after each test does not change the collected test content; it only removes the server from `wlp/usr/servers` as intended. Good.

2. The test framework does not fail a test when its server cannot be deleted due to file locking (ref: `LibertyFileManager.deleteLibertyDirectoryAndContents(machine, serverPath);`). Worst case the test inadvertently litters the locked server in `wlp/usr/servers`. Before these changes the test always littered 'TemplateServer` after the second test. I thought we should fix that.

File: dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/TemplateTests.java (hunk 1 above)
Comment (dazavala) [reply dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/TemplateTests.java:98]: Sam, before these changes the test left `TemplateServer` in `wlp/usr/servers` after `testCreateServerFromSpringBoot2`.  I thought we should fix that.

**Pull request merged**